src/utils.js

```diff
@@ -333,19 +353,23 @@ export function getTodaysPrestoMenu(menu) {
   return menu;
 }
 
-export function getTodaysVeglifeMenu(menu) {
+export function parseTodaysVeglifeMenu(menu) {
   const slovakDays = ['', 'PONDELOK', 'UTOROK', 'STREDA', 'ŠTVRTOK', 'PIATOK', 'SOBOTA'];
   const today = getDayForMenu();
-
-  return menu
+  menu = menu
```

Comment:
its generally a bad idea to modify the params
use another replace with regex like: `/+ Pestra.+$/`